planning/autoware_obstacle_stop_planner/schema/obstacle_stop_planner.schema.json

```
@@ -0,0 +1,322 @@
+        "predicted_object_filtering_threshold": {
+          "type": "number",
+          "description": "threshold for filtering predicted objects (valid only publish_obstacle_polygon true) [m]",
+          "default": "1.5"
+        },
+        "stop_planner": {
+          "type": "object",
+          "properties": {
+            "stop_position": {
+              "type": "object",
+              "properties": {
+                "max_longitudinal_margin": {
+                  "type": "number",
+                  "description": "stop margin distance from obstacle on the path [m]",
+                  "default": "5.0"
+                },
+                "max_longitudinal_margin_behind_goal": {
+                  "type": "number",
+                  "description": "stop margin distance from obstacle behind the goal on the path [m]",
+                  "default": "3.0"
+                },
+                "min_longitudinal_margin": {
+                  "type": "number",
+                  "description": "stop margin distance when any other stop point is inserted in stop margin [m]",
+                  "default": "2.0"
+                },
+                "hold_stop_margin_distance": {
+                  "type": "number",
+                  "description": "the ego keeps stopping if the ego is in this margin [m]",
+                  "default": "0.0"
+                }
+              },
+              "required": [
+                "max_longitudinal_margin",
+                "max_longitudinal_margin_behind_goal",
+                "min_longitudinal_margin",
+                "hold_stop_margin_distance"
+              ]
+            },
+            "detection_area": {
+              "type": "object",
+              "properties": {
+                "lateral_margin": {
+                  "type": "number",
+                  "description": "margin [m]",
+                  "default": "0.0"
+                },
+                "vehicle_lateral_margin": {
+                  "type": "number",
+                  "description": "margin of vehicle footprint [m]",
+                  "default": "0.0"
+                },
+                "pedestrian_lateral_margin": {
+                  "type": "number",
+                  "description": "margin of pedestrian footprint [m]",
+                  "default": "0.0"
+                },
+                "unknown_lateral_margin": {
+                  "type": "number",
+                  "description": "margin of unknown footprint [m]",
+                  "default": "0.0"
+                },
+                "step_length": {
+                  "type": "number",
+                  "description": "step length for pointcloud search range [m]",
+                  "default": "1.0"
+                },
+                "enable_stop_behind_goal_for_obstacle": {
+                  "type": "boolean",
+                  "description": "enable extend trajectory after goal lane for obstacle detection",
+                  "default": "True"
```

Could you make it to lower case?
It seems it causes some error when converting to markdown if false/true are capitalized.
```suggestion
                  "default": "true"
```